### ISSUE TITLE
musicexp.py:182 str not valid to int

### ISSUE THREAD

**Issue by mtarenskeen** [Philomelos/lilypond-musicxml2ly-dev#1]
in musicexp.py:182
the line

ly_dur = int(str)

gives me errors all the time when I try to use musicxml2ly current dev version. The errors disappear if I outcomment this line. Apparently something wrong here. If I look at the error messages in my terminal I see that sometimes str can be "8." or "1*2". These strings cannot be converted to int.
Also if I do

if str.isdigit(): ly_dur = int(str)

the error disappears. 

BTW: IMO it's a bad idea to give a string variable the name "str", a reserved keyword.

**Comment by Philomelos** [comment]
Yes, you're right.  I'll have a look at it as soon as I can.  Thanks for the report!

**Comment by Philomelos** [comment]
It was a regression I introduced when I was working on http://article.gmane.org/gmane.comp.gnu.lilypond.bugs/40460/match=musicxml2ly+tremolo+tag+notes+shorter+than+quarter.  For now I implemented your quick fix.  It should work ok for many files but we will have to find a solution for dotted notes and scaled durations in combination with tremolos.  (Philomelos/PUMTS/.../tremolo.xml works fine but then it doesn't contain any dotted notes or scaled durations...)

BTW: The use of "str" as a variable name is a relict of the LilyPond code.  I changed it in the class Duration to "dur_str" (duration string).
